### PR TITLE
Added linden portrait

### PR DESCRIPTION
**Pull request by drkm9743**

### CODE REVIEW

File: src/data/items.ts [hunk 1]
```diff
@@ -102,4 +102,10 @@ export const itemData: ItemInterface = {
 		tags: ['medical', 'cloth'],
 		description: 'Woven strip of material with absorbent barrier to prevent adhering to wounds.',
 	},
+	
+	'lindenportrait': {
+		label: 'Linden\'s Portrait',
+		tags: ['decoration', 'photo', 'furniture', 'australian'],
```
Comment (antond15) [inline src/data/items.ts]: Tags are listed [here](https://github.com/antond15/items/blob/main/src/data/tags.ts) so please use the available ones or create some new.
I would prefer using the `other` tag for this one.

File: src/data/items.ts [hunk 2]
```diff
@@ -102,4 +102,10 @@ export const itemData: ItemInterface = {
...
+		description: 'A portrait of the glorious Linden, who wouldn\'t want a photo of Linden?.',
```
Comment (antond15) [inline src/data/items.ts]: No dots after question marks please.

Comment (drkm9743) [reply src/data/items.ts]: bruh my bad kekw

File: src/data/items.ts (hunk 1 above)
Comment (drkm9743) [reply src/data/items.ts]: gotcha

**Comment by antond15** [comment]
Thanks!

**Pull request merged**